### PR TITLE
macOS-related fixes to the installer

### PR DESCRIPTION
**Pull request by gmerlino**
The installer does not work otherwise, at least under (recent) macOS versions

### CODE REVIEW

**Comment by wheatley** [comment]
@gmerlino thanks for the PR
we're looking at pushing this into our next release and was wondering if you'd like to join our discord?

**Comment by jackwalker-ripmfa** [comment]
@wheatley - should link to Discord in README where the other links to external BeEF resources are.

**Review by DeezyE — CHANGES_REQUESTED** [review]
thanks gmerlino, see my comments

File: .bundle/config [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
-BUNDLE_WITHOUT: "development"
-BUNDLE_WITH: "test:tests"
```
Comment (wheatley) [inline .bundle/config:3]: this needs to be removed due to release version 0.5.1.0 updating this file

**Review by wheatley — COMMENTED** [review]
added some comments for review
we'll be pushing this PR to the next release

**Comment by DeezyE** [comment]
@gmerlino are you available to address the above issues?

**Comment by wheatley** [comment]
Closing due to inactivity.

**Pull request closed without merging**